router/middleware/perm/perm.go

```diff
@@ -87,21 +88,40 @@ func MustSecretAdmin() gin.HandlerFunc {
 				return
 			}
 		case constants.SecretShared:
-			logrus.Debugf("Verifying user %s has 'admin' permissions for team %s/%s", u.GetName(), o, n)
-
-			perm, err := source.FromContext(c).TeamAccess(u, o, n)
-			if err != nil {
-				logrus.Errorf("unable to get user %s access level for team %s/%s: %v", u.GetName(), o, n, err)
+			if n == "*" && m == "GET" {
+				logrus.Debugf("Verifying user %s has 'admin' permissions for org %s", u.GetName(), o)
+
+				// TODO: Get list of teams user is a part of and filter by team in []
+				teams, err := source.FromContext(c).ListUsersTeamsForOrg(u, o)
+				if err != nil {
+					logrus.Errorf("unable to get users %s teams for org %s: %v", u.GetName(), o, err)
+				}
+
+				if len(teams) == 0 {
+					// nolint: lll // ignore long line length due to error message
```

Comment:
According to the linter, looks like we can remove this line:

https://github.com/go-vela/server/actions/runs/1120266414